src/scanners/javascript.js

```diff
@@ -45,10 +43,9 @@ export default class JavaScriptScanner {
     _ESLint = ESLint,
     _messages = messages,
     _ruleMapping = ESLINT_RULE_MAPPING,
-    // This tells ESLint where to expect the ESLint rules for addons-linter.
-    // Its default value is defined below. This property is mainly used for
-    // testing purposes.
-    _rulePaths = undefined,
+    // This property is used to inject additional custom eslint rules
+    // as part of tests.
+    _ruleModules = undefined,
```

Comment:
Why having a "modules" prefix and not simply `_rules`? (or `_additionalRules`)

Reply:
@willdurand No particular reason, I was trying to keep the same prefix as _ruleMapping, but I'm not really attached to `_ruleModules`,  `_rules` would be also totally fine by me.

I opted to _rules and rolled that change into the first patch part of this pull request.